src/components/plots/PlotContainer.jsx

```diff
@@ -83,19 +82,11 @@ const PlotContainer = (props) => {
   }, [config]);
 
   const onClickReset = () => {
-    onPlotReset();
     dispatch(resetPlotConfig(experimentId, plotUuid, plotType));
+    onPlotReset();
     setIsResetDisabled(true);
   };
 
-  if (!config) {
```

Comment:
This plot container is used in a lot of places. What happens now if config is not defined? Or is that case not possible anymore?

Reply:
Config generation is taken care of the parent components rendering this one and its not possible to not have a config and make this crash. I removed it because it was making the loading of multi plots impossible, because in that case the config is loaded in the component rendered by the `PlotContainer`, so it would never load the configs.